src/Provider/GraphHopper/GraphHopper.php

```diff
@@ -71,6 +76,11 @@ public function geocodeQuery(GeocodeQuery $query): Collection
 
         $url = sprintf(self::GEOCODE_ENDPOINT_URL, urlencode($address), $this->apiKey, $query->getLocale(), $query->getLimit());
 
+        $bounds = $query->getBounds();
+        if ($bounds) {
+            $url .= sprintf(self::GEOCODE_BOUNDS_PARAMS, $bounds->getWest(), $bounds->getSouth(), $bounds->getEast(), $bounds->getNorth());
```

Comment:
```suggestion
            $url .= sprintf('&bbox=%f,%f,%f,%f', $bounds->getWest(), $bounds->getSouth(), $bounds->getEast(), $bounds->getNorth());
```

`GEOCODE_BOUNDS_PARAMS` can be removed.

Reply:
done